packages/components/src/Label/_docs/Label.stories.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { Label } from "../index"
+import { LabelTypes } from "../types"
+
+const meta = {
+  title: "KAIO-staging/Label",
+  component: Label,
+  args: {
+    children: "I am Label",
+  },
+} satisfies Meta<typeof Label>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  parameters: {
+    docs: {
+      canvas: {
+        sourceState: "shown",
+      },
+    },
+  },
+}
+
+const InlineFormControl = (): JSX.Element => (
+  <span className="inline-block w-16 h-16 bg-gray-500"></span>
+)
+const BlockFormControl = (): JSX.Element => (
+  <span className="block w-200 h-16 bg-gray-500"></span>
+)
+
+export const InlineControls: Story = {
+  render: () => (
+    <Label labelText="Checkbox" labelType="checkbox">
+      <InlineFormControl />
+    </Label>
+  ),
+  parameters: {
+    docs: {
+      source: { type: "dynamic" },
+    },
+  },
+}
+
+/**
+ * By default, your label text will always sit to the right of the inline form control (ie. in RTL terms, the "end").
+ * Using the `labelPosition` you can flip the label text to sit to the left (ie. in RTL terms, the "start").
+ */
+
+export const Position: Story = {
+  render: () => (
+    <Label labelText="Checkbox" labelType="checkbox" labelPosition="start">
+      <InlineFormControl />
+    </Label>
+  ),
```

Comment:
You can use this format for all the stories which only have a single Label :)

```suggestion
  args: {
    labelText: "Checkbox",
    labelType: "checkbox",
    labelPosition: "start",
    children: <InlineFormControl />
  },
```

Reply:
You reminded me, this needed two examples to showcase difference between start and end